resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
I agree, so just keep the styling in `main.css` and 'ungeneralize' the button styling to an ID related styling, right? :)